src/themes/circuit.js

```diff
@@ -221,16 +221,19 @@ const breakpoints = {
 
 export const mq = createMediaQueries(breakpoints);
 
-export const transitions = {
-  default: `200ms ease-in-out`
-};
+export { default as transitions } from '../styles/transitions';
 
 // these values need to be properly trimmed/renamed as we go.
 export const zIndex = {
   default: 0,
   absolute: 1,
+  drawer: 10,
   popover: 20,
   select: 30,
   tooltip: 31,
   modal: 1000
 };
+
+export const palette = {
```

Comment:
What is this used for? I don't think we should expose something like this in the theme now. If people start using it for whatever reason it will introduce another breaking change once we sit down with designers to discuss missing aspects of Circuit UI.